ray-operator/controllers/ray/common/constant.go

```diff
@@ -125,6 +125,16 @@ const (
 
 	// Finalizers for RayJob
 	RayJobStopJobFinalizer = "ray.io/rayjob-finalizer"
+
+	DefaultAddressName                  = "address"
```

Comment:
We can add a comment to indicate that these are the parameter names for `ray start`: https://docs.ray.io/en/latest/cluster/cli.html?highlight=dashboard-host#ray-start.